packages/toast/toast-container.js

```diff
@@ -13,125 +13,135 @@ import { repeat } from 'lit/directives/repeat.js';
  */
 
 export class FabricToastContainer extends LitElement {
-    static styles = css`:host { display: block; }`;
-  
-    static properties = {
-        _toasts: { state: true },
-    };
-
-    constructor() {
-        super();
-        this._toasts = new Map();
+  static styles = css`
```

Comment:
Looks like formatting is changing to 2 spaces instead of 4. The general spacing rules at Finn has been 4 for as long as I've been at Finn but I don't actually have a preference. Should probably have a discussion about it if we are planning on changing it though.

Reply:
Is there some place I could check out to see what other rules we might like to stick to across all projects?
Here's what I found in `.prettierrc` in this repo:
```
{
  "proseWrap": "always",
  "singleQuote": true,
  "trailingComma": "all",
  "printWidth": 100,
  "tabWidth": 2,
  "overrides": [
    {
      "files": "*.css",
      "options": {
        "singleQuote": false
      }
    },
    {
      "files": "*.json",
      "options": {
        "tabWidth": 2
      }
    }
  ]
}
```

Reply:
Front page eslint
https://github.schibsted.io/finn/frontpage-podium/blob/master/.eslintrc

front page prettier 
https://github.schibsted.io/finn/frontpage-podium/blob/master/.prettierrc